<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BearingCalculator</class>
 <widget class="QDialog" name="BearingCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bearing Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="destLon"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Destination</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="radius"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Current</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="sourceLat"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="destLat"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="sourceLon"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Planet Radius (optional, km)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="output">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BearingCalculator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BearingCalculator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radius</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BearingCalculator</receiver>
   <slot>calculateBearingAndDistance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destLon</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BearingCalculator</receiver>
   <slot>calculateBearingAndDistance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destLat</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BearingCalculator</receiver>
   <slot>calculateBearingAndDistance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceLat</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BearingCalculator</receiver>
   <slot>calculateBearingAndDistance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceLon</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BearingCalculator</receiver>
   <slot>calculateBearingAndDistance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculateBearingAndDistance()</slot>
 </slots>
</ui>
